### PR TITLE
Documentation to create InitializerConfiguration persistent volume label controller in the cloud-controller-manager

### PR DESCRIPTION
**Pull request by rrati**
/milestone 1.8
/branch release-1.8

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5255)

### CODE REVIEW

File: docs/tasks/administer-cluster/running-cloud-controller.md [hunk 1]
```diff
@@ -32,3 +32,22 @@ To run `cloud-controller-manager`, add it to your existing Kubernetes cluster as
 The `kube-controller-manager` should not run any cloud-specific controllers, since the `cloud-controller-manager` takes over this responsibility. To prevent the `kube-controller-manager` from running cloud-specific controllers, you must set the `--cloud-provider` flag in `kube-controller-manager` to `external`.
 
 The `kube-apiserver` should not run the Persistent Volume Label admission controller either since the `cloud-controller-manager` takes over labeling persistent volumes.  To prevent the Persistent Volume Label admission plugin from running, make sure the `kube-apiserver` has a `--admission-control` flag with a value that does not include `PersistentVolumeLabel`.
+
+In order for the `cloud-controller-manager` to label persistent volumes, initializers will need need to be enabled and an InitializerConifguration will need to be added to the system.  Follow [these instructions](/docs/admin/extensible-admission-controllers.md#enable-initializers-alpha-feature) to enable initializers.  Use the following yaml to create the InitializerConfiguration:
+
+```shell
```
Comment (thockin) [inline docs/tasks/administer-cluster/running-cloud-controller.md]: s/shell/yaml ?

**Comment by steveperry-53** [comment]
@rrati, Suggested changes to the new text:

For the `cloud-controller-manager` to label persistent volumes, initializers need need to be enabled and an InitializerConifguration needs to be added to the system. Follow these instructions to enable initializers. Use the following YAML to create the InitializerConfiguration:

**Comment by steveperry-53** [comment]
@rrati, It's our convention to put YAML files in the documentation repo, and then use an {% include %} tag. See [Including code from another file](https://kubernetes.io/docs/home/contribute/write-new-topic/#including-code-from-another-file).
[Example](https://kubernetes.io/docs/tasks/configure-pod-container/assign-memory-resource/#specify-a-memory-request-and-a-memory-limit).

If you don't want to move or duplicate the YAML file, then use a full URL in the link. Otherwise, readers might not know where to find examples/xxx.

**Comment by rrati** [comment]
@steveperry-53 Is the include line correct?  When I try to look at it in github it doesn't include the text from the file but I don't see anything wrong with my line vs the example or other lines in the docs.

**Comment by steveperry-53** [comment]
@rrati, I don't see anything wrong with the include line. In the staged content (https://deploy-preview-5255--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-cluster/running-cloud-controller/), I do see the text of the YAML file. What's mysterious to me is that I don't see a title bar at the top of the YAML.

If it's OK with you, I'd like to go ahead and merge this PR. Then I can investigate the rendering of the YAML as a separate effort.

**Comment by steveperry-53** [comment]
@rrati, Now everything seems fine here:
https://deploy-preview-5255--kubernetes-io-vnext-staging.netlify.com/docs/tasks/administer-cluster/running-cloud-controller/

**Comment by rrati** [comment]
@steveperry-53 I'm fine getting this merged now

**Comment by rrati** [comment]
@chenopis What technical issue remain to be resolved?

**Comment by chenopis** [comment]
@rrati Looks like thockin's comment was addressed.

File: docs/tasks/administer-cluster/running-cloud-controller.md (hunk 1 above)
Comment (steveperry-53) [reply docs/tasks/administer-cluster/running-cloud-controller.md]: The YAML has been changed to an {% include %}.

**Pull request merged**